pyproject.toml

```diff
@@ -13,6 +13,7 @@ pytest = "^8.1.1"
 selenium = "^4.18.1"
 vcrpy = "^6.0.1"
 gunicorn = "^22.0.0"
+pytest-watch = "^4.2.0"
```

Comment:
Ah this annoyingly broke your hot-reloading in the dev container for me, I think because Docker/WSL don't play nicely together with `inotify`, but I'm happy to just ignore that here - removing the library resolved it but the pytest watch set up is nice 👍